main/main.go

```diff
@@ -0,0 +1,19 @@
+package main
```

Comment:
please put this package (main.go and both subcommands) into a directory called `cli/` or `patchutils/` instead of `main/`. This makes sure the compiled binary gets a somewhat saner name.